W3C.Domain/ChatService/ChatServiceClient.cs

```diff
@@ -29,6 +30,7 @@ public ChatServiceClient(IHttpClientFactory httpClientFactory)
         };
     }
 
+    [Trace]
```

Comment:
Probable want `[NoTrace]` on the auth params here?

Reply:
Oh yeah, good catch. Thanks!